traffic_server/plugins/astats_over_http/astats_over_http.c

```diff
@@ -538,9 +538,15 @@ void TSPluginInit(int argc, const char *argv[]) {
 	info.support_email = "justin@fp-x.com";
 	astatsLoad = time(NULL);
 
-	if (TSPluginRegister(&info) != TS_SUCCESS) {
-        TSError("Plugin registration failed. \n");
-	}
+	#if (TS_VERSION_NUMBER < 3000000)
+	  if (TSPluginRegister(TS_SDK_VERSION_2_0, &info) != TS_SUCCESS) {
+	#elif (TS_VERSION_NUMBER < 6000000)
+	  if (TSPluginRegister(TS_SDK_VERSION_3_0, &info) != TS_SUCCESS) {
+	#else
+	  if (TSPluginRegister(&info) != TS_SUCCESS) {
+	    TSError("Plugin registration failed. \n");
```

Comment:
Is this line intended to be inside the #if? It makes the first two options expand to empty tests, which seems wrong.